user.go

```diff
@@ -31,12 +31,12 @@ type Officership struct {
 
 // Photo represents a photo of a user.
 type Photo struct {
-	PhotoId      uint   `json:"photoid"`
+	PhotoID      uint   `json:"photoid"`
```

Comment:
The issue with that approach is that you'd end up being inconsistent when dealing with tables (like officerships) that have multiple IDs in them. It's perhaps not ideal, but I'd suggest NameofThingID where ever possible